lunabot_behavior/plunge.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, excavation_publisher: rospy.Publisher = None, lin_act_publish
         # TODO: check and test this value
         self.LOWERING_TIME = 23 #In seconds, how long it takes to lower the linear actuators 90% of the way
 
-        self.EXCAVATION_SPEED = 127
+        self.EXCAVATION_SPEED = 1000 #RPM
         self.LIN_ACT_SPEED = -110
 
         self.is_sim = rospy.get_param("is_sim")
```

Comment:
Github won't let me leave a comment in the right place, but two lines down (line 54) exc_message is not updated I don't think

Reply:
Updated in fix threshold commit